src/user-event/scroll/scrollTo.ts

```diff
@@ -0,0 +1,139 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { UserEventInstance } from '../setup';
+import { EventBuilder } from '../event-builder';
+import { ErrorWithStack } from '../../helpers/errors';
+import { isHostScrollView } from '../../helpers/host-component-names';
+import { dispatchEvent } from '../utils';
+import { ContentOffset } from '../event-builder/scroll';
+import {
+  createHorizontalScrollSteps,
+  createVerticalScrollSteps,
+} from './utils';
+import { getElementScrollOffset, setElementScrollOffset } from './state';
+
+export interface VerticalScrollToOptions {
+  y: number | number[];
```

Comment:
nit: I like adding x?: never in these type of cases so that I can do if(options.x) rather than if('x' in options). It's a more usual syntax and if the prop name changes then there will be an error whereas with if('x' in options) you'll never get an error

Reply:
Changed!